app/services/vot/parser.rb

```diff
@@ -86,6 +88,7 @@ def expand_components_with_initial_components(initial_components)
         identity_proofing?: requirement_list.include?(:identity_proofing),
         biometric_comparison?: requirement_list.include?(:biometric_comparison),
         ialmax?: requirement_list.include?(:ialmax),
+        enhanced_ipp?: requirement_list.include?(:enhanced_ipp)
```

Comment:
I bet the linter is going to want a trailing comma

Reply:
that's what i'm seeing in gitlab